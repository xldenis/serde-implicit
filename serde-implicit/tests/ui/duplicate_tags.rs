#[derive(serde_implicit_proc::Deserialize)]
enum MultiTagFields {
    DoubleTagged {
        #[serde_implicit(tag)]
        primary_tag: String,
        #[serde_implicit(tag)]
        secondary_tag: bool,
        value: u32,
    },
    SingleTagged {
        #[serde_implicit(tag)]
        only_tag: u32,
        value: String,
    },
}

fn main() {}
