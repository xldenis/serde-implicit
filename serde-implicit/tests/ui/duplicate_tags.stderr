error: duplicate `#[tag]` annotations found, only one field can be tagged
 --> tests/ui/duplicate_tags.rs:3:18
  |
3 |       DoubleTagged {
  |  __________________^
4 | |         #[tag]
5 | |         primary_tag: String,
6 | |         #[tag]
7 | |         secondary_tag: bool,
8 | |         value: u32,
9 | |     },
  | |_____^
